### ISSUE TITLE
Screen Blinking while playing Audio

### ISSUE THREAD

**Issue by krishnaTeja5059** [AppLozic/ApplozicSwift#209]
**Details:**

- Device : iPhone 6
- OS version: 12.2
- Xcode version: 10.2.1

 **While playing Audio , For every second Screen was blinking.**

[![ezgif.com-video-to-gif0da69b3b70186280.gif](https://media.giphy.com/media/LQ2G87zxjXogLW2mg2/giphy.gif)

**Comment by mukeshthawani** [comment]
Thanks for sharing the information. Can you share the ApplozicSwift version you are using? We tried with the latest version but couldn't reproduce this issue.

**Comment by sureshMopidevi** [comment]
Thanks for your quick response,
`Pod version: 2.6.0`

**Comment by mukeshthawani** [comment]
@sureshMopidevi we tried with 2.6.0 but we were not able to reproduce the issue. Is it possible for you to create a user and a conversation where we can reproduce this issue? You can share those details at support@applozic.com.

Also, if possible try our sample app and see if you can reproduce the issue.

**Comment by krishnaTeja5059** [comment]
I observed that, This issue happening randomly. I have tested the **_Demo app_** in repo and found same issue for one time. i tried to reproduce it but  unable to track it . 
**Observations**
- Flickering happening when visible cells contains `generic cards (rich messages)`.
- when i clicked on play button, View went up for some times. 
- While playing audio, the track duration was not accurate.
it shows current duration and total duration for each second
